### PR TITLE
Updating signal samples with the split out values

### PR DESCRIPTION
**Pull request by scarletnorberg**

### CODE REVIEW

**Comment by zhenbinwu** [comment]
From a quick glance, I don't spot issues.

File: sampleSets_PostProcessed_2017.cfg [hunk 1]
```diff
@@ -169,8 +169,13 @@ ZZZ_2017, /eos/uscms/store/user/lpcsusyhad/Stop_production/Fall17_94X_v2_NanAOD_
 WZG_2017, /eos/uscms/store/user/lpcsusyhad/Stop_production/Fall17_94X_v2_NanAOD_MC/PostProcessed_15Jan2019_v6, WZG_2017.txt, Events, 0.04123, 920436, 79564, 1.0
 WWG_2017, /eos/uscms/store/user/lpcsusyhad/Stop_production/Fall17_94X_v2_NanAOD_MC/PostProcessed_15Jan2019_v6p1, WWG_2017.txt, Events, 0.2147, 879311, 82589, 1.0
 
+####POWHEG
+TTToHadronic_powheg_2017, /eos/uscms/store/user/lpcsusyhad/Stop_production/Fall17_94X_v2_NanAOD_MC/PostProcessed_18Apr2019_v6p2/, TTToHadronic_powheg_2017.txt, Events, 380.095, 129736822, 525618, 1.0
```
Comment (zhenbinwu) [inline sampleSets_PostProcessed_2017.cfg:173]: Why do you add POWHEG in 2017/2018, but not in 2016?

**Comment by zhenbinwu** [comment]
For the signal, I think @ahenckel should review as I can't tell the details

**Comment by ahenckel** [comment]
Looks good to me.

**Comment by zhenbinwu** [comment]
Ok, I will add back the POWHEG 2016 manually. Merging this PR so that I can start

**Pull request merged**